### DISCUSSION TITLE
loss functıon

### DISCUSSION THREAD

**Discussion by ayusakoc** [MIC-DKFZ/nnUNet#2978] [Ideas]
In tumor training, since the tumor is relatively small compared to the organ, would it be logical to replace the default loss function (dice+crossentropy) with (dice+focalLoss) only in tumor training?
